### DISCUSSION TITLE
CoPilot terminal blank/not working in latest Visual Studio Code

### DISCUSSION THREAD

**Discussion by JakeRuth** [community/community#38029] [Copilot Conversations]
GitHub Copilot version: v1.56.7152
Visual Studio Code version: 1.72.2
Operating System: Ubuntu 20.x

* I created a paid Github account/connected a credit card
* Installed copilot in Visual Studio Code
* Logged in/went through auth process with Github
* CoPilot claims it's installed, but isn't doing anything
* Output is blank, I never see any out.
* I tried uninstalling and reinstalling and force quitting/reopening Visual Studio Code a few times, no luck
* I checked a lot of other threads about this, some show connection errors, others show authentication issues. I cannot find an existing issue that shows my issue where the terminal output is empty, it says it's installed, but it doesn't do anything
* Any help is appreciate, thank you

![Screenshot from 2022-11-01 21-33-47](https://user-images.githubusercontent.com/7349551/199374407-6beed5e7-8ab1-4685-87c8-94706cc425a3.png)
![Screenshot from 2022-11-01 21-33-36](https://user-images.githubusercontent.com/7349551/199374410-db5792a4-4b9f-4b53-90a9-7b1562c3a9bc.png)
![Screenshot from 2022-11-01 21-39-17](https://user-images.githubusercontent.com/7349551/199374511-1a4312e5-7508-46bb-8f0e-81b72af8bf64.png)

**Comment by davecheney** [comment]
@JakeRuth could you try two things for me please.

1. Double check that you have VSCode, not VScodium or OSSCode installed (it can be tricky on linux because the latter are sometimes packaged by linux os vendors)
2.  <img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
   Please try signing out, copilot will prompt you to sign back in.

**Reply by JakeRuth** [reply]
> @JakeRuth could you try two things for me please.
> 
> 1. Double check that you have VSCode, not VScodium or OSSCode installed (it can be tricky on linux because the latter are sometimes packaged by linux os vendors)
> 2. <img alt="Screen Shot 2021-11-30 at 8 03 57 am" width="475" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
> 
> Please try signing out, copilot will prompt you to sign back in.

Hi Dave, I appreciate you replying thank you.

I double checked my install, I'm confident it's Visual Studio Code, but could be confused. Here's an image when I click "About" from the Help menu:
![Screenshot from 2022-11-02 13-12-38](https://user-images.githubusercontent.com/7349551/199557001-2210fc61-9eb1-4a9d-99ac-646c18c80f43.png)

I also followed your instruction to log out. However it seems I'm still in a weird state. I am not getting prompted to log back in. I also see that the plugin is enabled/seems to think it's still activated. But in the same state I originally described where the output is blank/nothing is happening.
![Screenshot from 2022-11-02 13-13-43](https://user-images.githubusercontent.com/7349551/199556945-beab5979-c64f-4c66-bcb3-77ac7f6ea2be.png)

It's the "morning" for me now, so I'll be able to reply faster (I was about to sleep when you replied last night!)

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by JakeRuth** [reply]
> Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

Highly unlikely, I'm on a personal laptop running my own startup so there's no "employer" and I've never had these issues before. Is there an easy way to check this just to confirm?

The frustrating part of this is I can't find logs or output anywhere. It's as if the copilot extension is muted and doesn't even have the ability to print to the console

**Reply by davecheney** [reply]
Thanks for replying. Sorry, I had to ask, this is a very common issue for customers who are being restrictive networks. 

I’d suggest trying the VSCode version from their website, not the snap install.

**Reply by JakeRuth** [reply]
> Thanks for replying. Sorry, I had to ask, this is a very common issue for customers who are being restrictive networks.
> 
> I’d suggest trying the VSCode version from their website, not the snap install.

Dave thank you for replying! I feel helping :''') I really want to use copilot and I'm excited about it, but I keep seeing no console output, and every UI in this IDE tells me things should be fine an dandy.

I went ahead and downloaded the right version from their site, still the same behavior and issue :(

Idk if this is an option, but would screen sharing on this help in a quick vcall? I'm willing to do anything to get this to work.

**Reply by davecheney** [reply]
I;'m sorry, that's not a level of support i can provide. I think it is likely this is an issue with your local workstation. I'd encourage you to try uninstalling other extensions and see if that resolves the conflict.

**Reply by JakeRuth** [reply]
> I;'m sorry, that's not a level of support i can provide. I think it is likely this is an issue with your local workstation. I'd encourage you to try uninstalling other extensions and see if that resolves the conflict.

That's fair. I appreciate your help thanks Dave

**Reply by ITatRPR** [reply]
For what it's worth, had the same issue. This is what helped - 

Bottom left, sign out, close the VScode.
Open VScode and sign in again to git.

Worth a try.
Andy
